media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -1472,12 +1472,21 @@ textarea:focus {
     display: none;
 }
 
+.mozillians-profile.friends {
+    background: #fff url('/media/img/contribute/mozillians-firefoxfriends.png') center center no-repeat;
+    background-size: 100%;
+}
+
+.mozillians-profile.oneandone {
```

Comment:
In the states, a one and one is a double shot of espresso split into two demitasses, with cappuccino type milk added to one of them.

Anyway, maybe change the class name to `oneanddone`? Doesn't really matter. Just wanted to talk about coffee for a minute. :stuck_out_tongue_closed_eyes: